packages/@okta/vuepress-site/docs/reference/api/myaccount/index.md

```diff
@@ -0,0 +1,470 @@
+        },
+        "user": {
+            "href": "https://${yourOktaDomain}/api/v1/myaccount"
+        }
+    },
+    "createdAt": "2020-01-14T20:05:32.000Z",
+    "modifiedAt": "2020-10-13T03:17:09.000Z",
+    "profile": {
+        "customBoolean": null,
+        "foo": "bar",
+        "login": "example@ex.ample.com",
+        "mobilePhone": null,
+        "customInteger": null
+    }
+}
+```
+
+
+### Update My User Profile
+
+<ApiOperation method="put" url="/api/v1/myaccount/directoryProfile" />
+
+Updates the caller's MyUserProfile.
+
+> **Note:** This API differs from the the existing [Users API](/docs/reference/users/) in that only PUT is supported.  This API also does not support partial update.  All values returned from fetching MyUserProfile must be passed to this API, or the update will not pass validation.  This applies even if the omitted schema property is optional. To unset an optional property, explicitly pass the property with a value of `null`.
+
+#### Request path parameters
+
+N/A
+
+#### Request query parameters
+
+N/A
+
+#### Request body
+
+This API requires the `profile` property of a [MyUserProfile](#myuserprofile-object) as its request body.
+
+| Property | Type                     | Description                          |
+| -------- | -------------------------|--------------------------------------|
+| profile  | A map of key-value pairs | The properties defined in the schema |
+
+#### Response body
+
+Returns the result of applying the update, as if the caller had invoked the GET MyUserProfile operation.
+
+#### Usage example
+
+##### Request
+
+This request would update the user profile of the caller to have exactly the values specified.
+
+```bash
+curl -XPUT 'https://${yourOktaDomain}/api/v1/myaccount/directoryProfile' -H 'Authorization: SSWS {token}' -H 'Content-Type: application/json' --data '{
+>     "profile": {
+>         "customBoolean": false,
+>         "foo": "bar",
+>         "login": "dayton.williams@okta.com",
+>         "notFive": 5,
+>         "customInteger": null
+>     }
+> }'
+```
+
+##### Response
+
+```json
+{
+    "_links": {
+        "describedBy": {
+            "href": "https://${yourOktaDomain}/api/v1/myaccount/profile/schema"
+        },
+        "self": {
+            "href": "https://${yourOktaDomain}/api/v1/myaccount/directoryProfile"
+        },
+        "user": {
+            "href": "https://${yourOktaDomain}/api/v1/myaccount"
+        }
+    },
+    "createdAt": "2020-01-14T20:05:32.000Z",
+    "modifiedAt": "2020-11-03T06:01:13.000Z",
+    "profile": {
+        "customBoolean": false,
+        "foo": "bar",
+        "login": "example@ex.ample.com",
+        "mobilePhone": null,
+        "customInteger": 5
+    }
+}
+```
+
+
+
+
+## MyAccount API objects
+
+### Me object
+
+#### Me properties
+
+The Me object has several properties:
+
+Note: Some returned links point to operations planned but not yet implemented.  Currently supported links are to directoryProfile and self.
```

Comment:
Again, why would we return invalid links?

Reply:
We should remove the links if they are not implemented

Reply:
See previous comment.  Removed.